README.md

```diff
@@ -183,17 +183,17 @@ Passing options to the pass can be done using either an array or the
 SpanOptions wrapper object. The following keys are valid:
 
 - `start_time` is a float, int or `\DateTime` representing a timestamp with arbitrary precision.
-- `child_of` is an object of type `OpenTracing\SpanContext` or `OpenTracing\Span`.
-- `references` is an array of `OpenTracing\SpanReference`. 
+- `references` is an array of `OpenTracing\Reference`. 
 - `tags` is an array with string keys and scalar values that represent OpenTracing tags.
 
 ```php
 <?php
 
+use OpenTracing\Reference;
 use OpenTracing\SpanOptions;
 
 $span = $tracer->createSpan('my_span', SpanOptions::create([
-    'child_of' => $parentContext,
+    'references' => Reference::create(Reference::CHILD_OF, $spanContext),
```

Comment:
it many languages we chose to explicitly support `child_of` optional parameter because it is used in 99% of all instrumentation cases.